Makefile

```diff
@@ -27,16 +28,25 @@ hashicorp-down:
 
 networks:
 	@docker network create --driver=bridge hashicorp || true
+	@docker network create --driver=bridge postgres || true
```

Comment:
Do we need a postgres network? Ideally we should have a qkm network where we attach mandatory services like postgres to. Only in case of optional deps is where we do the otherway around

Reply:
modified